src/kvstore/raftex/RaftexService.h

```diff
@@ -49,6 +46,23 @@ class RaftexService : public cpp2::RaftexServiceSvIf {
     void removePartition(std::shared_ptr<RaftPart> part);
 
 private:
+    void initThriftServer(std::shared_ptr<folly::IOThreadPoolExecutor> pool, uint16_t port = 0);
+    bool setup();
+    void serve();
+
+    enum RaftServiceStatus{
```

Comment:
Do we really need the complicated status here?

Reply:
yes, I will fix it, thx.